source/user_guide/case_auditing.rst

```diff
@@ -0,0 +1,138 @@
+.. currentmodule:: howso.engine
+
+
+Case Auditing
```

Comment:
should this guide really be called Case Auditing??

Strikes me as closer to like "Understanding Predictions" or "Auditing Predictions"

Auditing cases makes me think of finding cases that are illegitimate or biased